lib/wallet/wallet.js

```diff
@@ -5551,24 +5526,22 @@ class Wallet extends EventEmitter {
    * @param {String | undefined} cacheKey
    */
 
-   clearCache(cacheName, cacheKey) {
-      let cache;
-      if(cacheName === 'bid') {
-        cache = this.sendBidResults;
-      } else if (cacheName === 'open') {
-        cache = this.sendOpenResults;
-      } else if (cacheName === 'update') {
-        cache = this.sendUpdateResults;
-      } else {
-        throw new Error(`invalid cache name: ${cacheName}`);
-      }
+  clearCache(cacheName, cacheKey) {
+    let cache;
+    if(cacheName === 'bid') {
```

Comment:
nit missing space `if (cacheName)`

Also seems like there's no more open cache? Why are there any caches left here? I would have expected them all to be moved to the plugin can you explain

Reply:
There was a reason i specifically let them in wallet.js but cant remember it, and i dont see a reason why we moved them to plugin now, i am moving it. and yes I did removed the opencache thinking opencache is not needed anymore but, i will add it back

Reply:
Cache functionality moved into plugin